src/backend/services/logger/middleware.py

```diff
@@ -16,6 +14,13 @@ async def dispatch(self, request: Request, call_next):
 
         ctx = get_context(request)
```

Comment:
Is context always not None?

Reply:
Yep, it's either the existing context or a newly instantiated one, never None